CONTRIBUTING.md

```diff
@@ -23,7 +23,57 @@ use GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
 
+## KPT Packages
+
+In order to maintain consistency accross kpt packages, we strive to maintain the following standards.
+
+### Generic
+
+1. Google's License text is included at the beginning of each resource file
+1. Avoid blank lines
+1. `metadata.name` include the `project-id` prefix only when K8S uniqueness is required
+1. `spec.resourceID` include the `project-id` prefix only when GCP uniqueness is required
+1. Resource's fields/subfields are ordered as per resource's schema
+1. Resource's reference should use `name` instead of `external` as much as possible
+
+### Comments
+
+1. A comment line is included prior to every resources definition
+1. The resource description field has the same information as the comment line at the beginning of the resource definition
+1. Avoid comments on same line as a field
+1. `kpt-merge` comment should be removed
+1. Include a comment for every setter in `setters.yaml`
+
+### Annotations
+
+1. Include
+
+    - `depends-on` for every resource
+
+1. Remove :
+
+    - `cnrm.cloud.google.com/blueprint: 'kpt-fn'`
+
+### Markdown
+
+1. `readme.md` is generated using the command below. Replace `REPO_PACKAGE_DIR` and `LOCAL_PACKAGE_DIR` accordingly.
+
+    ```shell
+    REPO_URL=https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/REPO_PACKAGE_DIR
+    kpt fn eval -i generate-kpt-pkg-docs:unstable --mount type=bind,src="LOCAL_PACKAGE_DIR",dst="/tmp",rw=true -- readme-path=/tmp/README.md repo-path=$REPO_URL
```

Comment:
remove space after double minus -- readme-path=

Reply:
somehow, the space seems to make this command works. I actually always used it exactly like that

Reply:
It as per the documentation, `$ kpt fn eval -i generate-kpt-pkg-docs:unstable --include-meta-resources \
--mount type=bind,src="$(pwd)",dst="/tmp",rw=true -- readme-path=/tmp/README.md`

https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/go/generate-kpt-pkg-docs

Reply:
Just wondering if this to is missing a `$` - src="LOCAL_PACKAGE_DIR" -> src="$LOCAL_PACKAGE_DIR" OR src="${LOCAL_PACKAGE_DIR}"

Reply:
From my understanding, everything before the lonely `--` are arguments for `kpt fn eval`, everything after are for the function itself (`generate-kpt-pkg-docs`)

This has worked in the past with WSL (it runs from the repo's root and the `--as-current-user` can avoid some permission issues):
```bash
# set PKG_PATH appropriately and confirm a 'README.md' exists with HTML markers in PKG_PATH
# https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/go/generate-kpt-pkg-docs

PKG_PATH='solutions/gatekeeper-policies'
REPO_URL='https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git'

# run from repo root, uses current user, mounting PKG_PATH to /tmp
kpt fn eval -i generate-kpt-pkg-docs:unstable "${PKG_PATH}" \
    --as-current-user \
    --mount type="bind",src="$PWD/${PKG_PATH}",dst="/tmp",rw="true" -- \
    readme-path="/tmp/README.md" \
    repo-path="${REPO_URL}" \
    pkg-name="/${PKG_PATH}"
```

Reply:
Sounds good, normally I run each command myself to test - not this time on my part.  I appreciate the investigation and heads up on the standalone --